src/pages/patientView/SampleManager.tsx

```diff
@@ -279,6 +292,53 @@ class SampleManager {
         }
         return '';
     }
+
+    getClinicalAttributeList(samples: Array<ClinicalDataBySampleId>) {
```

Comment:
this is essentially a static function since it does not make use of instance (this).  You could make it a helper function.   You should also improve the name:  getClinicalAttributeListForSamples

Reply:
Fixed